janus_core/src/test_util/kubernetes.rs

```diff
@@ -90,6 +145,22 @@ impl Drop for EphemeralCluster {
     }
 }
 
+/// A guard on a running `kubectl port-forward` process. The process will be killed and reaped when
+/// the guard is dropped.
+pub struct PortForwardGuard {
```

Comment:
Name this a `PortForward` since that's what it is -- the fact that it implements a "drop-guard" pattern is somewhat an implementation detail. (e.g., we don't call the type `ArcGuard`, we call it `Arc`.)